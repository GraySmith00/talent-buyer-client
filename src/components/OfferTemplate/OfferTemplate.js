import React, { Component } from 'react';
import Nav from '../Nav/Nav';

import './OfferTemplate.css';

class OfferTemplate extends Component {
  render() {
    return (
      <div className="offer-template">
        <Nav />
        <h1>Offer Template</h1>
      </div>
    );
  }
}

export default OfferTemplate;
