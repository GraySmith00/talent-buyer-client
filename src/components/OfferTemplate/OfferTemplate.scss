.offer-template {
  display: flex;
  background-color: #21222b;
  color: #fff;
  overflow: scroll;
}
