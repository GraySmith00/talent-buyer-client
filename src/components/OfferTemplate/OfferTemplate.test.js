import React from 'react';
import { shallow } from 'enzyme';

import OfferTemplate from './OfferTemplate';

describe('OfferTemplate component', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<OfferTemplate />);
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
