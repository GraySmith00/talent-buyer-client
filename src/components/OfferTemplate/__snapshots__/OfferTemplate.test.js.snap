// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OfferTemplate component should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <OfferTemplate />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Nav />,
        <h1>
          Offer Template
</h1>,
      ],
      "className": "offer-template",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Offer Template",
        },
        "ref": null,
        "rendered": "Offer Template",
        "type": "h1",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Nav />,
          <h1>
            Offer Template
</h1>,
        ],
        "className": "offer-template",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Offer Template",
          },
          "ref": null,
          "rendered": "Offer Template",
          "type": "h1",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
